### PR TITLE
Deleted json tags, added gorm tags

### PR DESCRIPTION
**Pull request by bokimilinkovic**
https://trello.com/c/4J7eoAHO/66-remove-json-tags-from-book-model

### CODE REVIEW

File: domain/model/book.go [hunk 1]
```diff
@@ -3,14 +3,14 @@ package model
...
-	Year          string    `json:"publish_date"`
-	CreatedAt     time.Time `json:created_at`
-	UpdatedAt     time.Time `json:updated_at`
+	Id            int
+	Title         string
+	Author        string
+	Isbn          string    `gorm:"colum:isbn_10"`
+	Isbn13        string    `gorm:"column:isbn_13"`
+	OpenLibraryId string    `gorm:"column:open_library_id"`
+	CoverId       string    `gorm:"column:cover_id"`
```
Comment (sfqi) [inline domain/model/book.go]: I don't think we need every one of those tags. See http://gorm.io/docs/conventions.html#Snake-Case-Column-Name

File: domain/model/book.go [hunk 2]
```diff
@@ -3,14 +3,14 @@ package model
...
-	CreatedAt     time.Time `json:created_at`
-	UpdatedAt     time.Time `json:updated_at`
+	Id            int
+	Title         string
+	Author        string
+	Isbn          string
+	Isbn13        string
+	OpenLibraryId string
+	CoverId       string
+	Year          string    `gorm:"-"`
```
Comment (sfqi) [inline domain/model/book.go]: Don't we have a `year` column?

Comment (bokimilinkovic) [reply domain/model/book.go]: I didn't get what we were doing with publish_date in domain model previously .. but I see now stefan changed that to 'year' , so okay i will change it.

Comment (bokimilinkovic) [reply domain/model/book.go]: but year is int in db scheme as i recall, how can this be string then?

Comment (sfqi) [reply domain/model/book.go]: You can create a trello card for that (e.g. "modify `model.Book.Year` type to int")

File: domain/model/book.go [hunk 3]
```diff
@@ -3,14 +3,14 @@ package model
...
-	CreatedAt     time.Time `json:created_at`
-	UpdatedAt     time.Time `json:updated_at`
+	Id            int
+	Title         string
+	Author        string
+	Isbn          string
+	Isbn13        string
+	OpenLibraryId string
+	CoverId       string
+	Year          string    `gorm:"year"`
```
Comment (sfqi) [inline domain/model/book.go]: I don't think this is needed. If the attribute is called `Yead`, gorm automatically maps it to column `year`

**Pull request merged**